ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -8,28 +8,35 @@
   }
 
   &__account-name {
+    display: inline;
     font-weight: bold;
     font-size: 14px;
     line-height: 20px;
   }
 
+  %account-status-typography {
+    font-size: 12px;
+    line-height: 17px;
+    padding-top: 4px;
+  }
+
   &__account-status {
+    @extend %account-status-typography;
+    display: inline;
     color: $Grey-500;
   }
 
   &__account-status-link {
+    @extend %account-status-typography;
+    display: block;
+    color: $Grey-500;
```

Comment:
Nit: this has no effect - it's directly overwritten by the block below where it's set to `$curious-blue`, which is the colour it is in the designs anyway.
```suggestion
```